app/controllers/users_controller.rb

```diff
@@ -1,3 +1,32 @@
 class UsersController < ApplicationController
-  def new; end
+
+  USER_ATTRIBUTES = User::USER_ATTRIBUTES
+
+  def show
+    @user = User.find_by(params[:id])
```

Comment:
truyền key vào method `find_by`
```suggestion
    @user = User.find_by id: params[:id]
```